spec/http_request_spec.rb

```diff
@@ -17,7 +17,7 @@
     let(:http_get_mock) { double(basic_auth: true) }
 
     before do
-      allow(Net::HTTP).to receive(:start).with(uri.host).and_return(http_mock)
+      allow(Net::HTTP).to receive(:start).with(uri.host, { use_ssl: true}).and_return(http_mock)
```

Comment:
same here

Reply:
I think this slipped when running rubocop...
![image](https://user-images.githubusercontent.com/1621114/35336892-0cd867c2-011a-11e8-862c-b54923d9e948.png)